director/sort.go

```diff
@@ -156,33 +156,40 @@ func getLatLong(ctx context.Context, addr netip.Addr) (lat float64, long float64
 		return override.Lat, override.Long, nil
 	}
 
+	labels := prometheus.Labels{
+		"network": "",
+		"source":  "",
+		"project": "",
+	}
+
 	network, ok := utils.ApplyIPMask(addr.String())
 	if !ok {
 		log.Warningf("Failed to apply IP mask to address %s", ip.String())
+	} else {
+		labels["network"] = network
 	}
 
 	project, ok := ctx.Value(ProjectContextKey{}).(string)
 	if !ok || project == "" {
 		log.Warningf("Failed to get project from context")
-		project = "unknown"
+		labels["project"] = "unknown"
+		labels["network"] = "unknown"
```

Comment:
Can you move setting the unknown network into the `!ok` check after running `utils.ApplyIPMask`, or is the intended behavior that we'd never set the network if there's no project?